### PR TITLE
[docs] Added tables instead of images at versioning modules

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Added tables instead of images at versioning modules.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
type: fix 
summary: Added tables instead of images at versioning modules.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Lada7878. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 1]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
 - изменение **Minor-версии** (например, c `0.0.1` на `0.1.0`) — добавление новой функции;
 - изменение **Major-версии** (например, c `0.0.1` на `1.0.0`) — добавление функции, которая кардинально меняет возможности модуля; масштабное изменение интерфейса или завершение крупного этапа работы.
 
-Перед номером версии в теге git и контейнере registry **всегда** добавляется буква "v". Примеры: `v0.0.73`, `v1.0.0`.
+Перед номером версии в теге git и контейнере registry **всегда** добавляется буква "v". Например: `v0.0.73`, `v1.0.0`.
 
 ## Каналы обновлений
 
-Версия модуля при публикации должна *перемещаться* по [каналам обновлений](../../deckhouse-release-channels.html) от менее стабильного к более стабильному: `Alpha` -> `Beta` -> `EarlyAccess` -> `Stable` -> `RockSolid`.
+Версия модуля при публикации должна оставаться неизменной на всех [каналах обновлений](../../deckhouse-release-channels.html) от менее стабильного к более стабильному: `Alpha` → `Beta` → `EarlyAccess` → `Stable` → `RockSolid`.
```
Comment (z9r5) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:18]: @mortelumina98 тут не нужно про неизменность версии писать. Тут речь про то, что разработчик должен публиковать версию с учетом стабильности каналом. Например, надо сначала на ALpha, потом на Beta, а не наоборот. И так по возрастанию стабильности. Возможно всю фразу надо переформулировать, раз она вызывает вопросы.

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 2]
```diff
@@ -71,8 +182,9 @@ lang: ru
...
+Это может понадобиться, например, при переименовании или перемещении параметра в новой версии OpenAPI-спецификации.
 
-Рекомендации, которых стоит придерживаться при выходе новой версии *CustomResourceDefinition* (CRD):
-* Предыдущим версиям проставлять параметр `deprecated: true` (читайте подробнее в документации [Kubernetes](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-deprecation)).
+При выходе новой версии *CustomResourceDefinition* (CRD) используйте следующие рекомендации:
+* Предыдущим версиям проставлять параметр `deprecated: true`, подробнее в документации [Kubernetes](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#version-deprecation).
 * Версию, в которой данные хранятся внутри etcd ([storage-версия](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version)), менять не ранее чем через два месяца после выхода новой версии.
```
Comment (z9r5) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:190]:
```suggestion
* Версию, в которой данные хранятся внутри etcd ([storage-версия](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version)), меняйте не ранее чем через два месяца после выхода новой версии.
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 3]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
 ## Как понять, насколько модуль стабилен?
 
 В зависимости от этапа жизненного цикла модуля и канала обновлений, из которого была установлена версия модуля, общая стабильность может быть определена в соответствии со следующей таблицей:
 
-![Module_Stability](../../images/module-development/module_stability.png)
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Этапы модулей</title>
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:44]:
```suggestion
    <title>Статус модуля</title>
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 4]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
+
+</body>
+</html>
 
 Выводы:
 - Модуль на стадии `Experimental` в канале `Stable` не рекомендуется использовать в production-средах.
 - Модуль на стадии `GA` в канале `Alpha` также не рекомендуется использовать в production-средах.
 - Для production-сред подходят только модули, находящиеся на стадии `GA`, установленные из каналов `EarlyAccess`, `Stable`, или `RockSolid`.
-- Модули, находящиеся на стадии `Deprecated`, рекомендуется заменить согласно рекомендациям в документации.
+- Модули, находящиеся на стадии `Deprecated`, рекомендуется заменить.
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:157]:
```suggestion
- Модули, находящиеся в статусе`Deprecated`, рекомендуется заменить.
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 5]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
+            <td class="grey" style="text-align:center; vertical-align: middle;">Необходимо выводить из использования</td>
+        </tr>
+    </tbody>
+</table>
+
+</body>
+</html>
 
 Выводы:
 - Модуль на стадии `Experimental` в канале `Stable` не рекомендуется использовать в production-средах.
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:154]:
```suggestion
- Модуль в статусе`Experimental` в канале `Stable` не рекомендуется использовать в production-средах.
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 6]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
 - Модуль на стадии `GA` в канале `Alpha` также не рекомендуется использовать в production-средах.
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:155]:
```suggestion
- Модуль в статусе `GA` в канале `Alpha` также не рекомендуется использовать в production-средах.
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 7]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
 - Для production-сред подходят только модули, находящиеся на стадии `GA`, установленные из каналов `EarlyAccess`, `Stable`, или `RockSolid`.
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:156]:
```suggestion
- Для production-сред подходят только модули, находящиеся в статусе `GA`, установленные из каналов `EarlyAccess`, `Stable`, или `RockSolid`.
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 8]
```diff
@@ -6,44 +6,155 @@ lang: ru
...
+- изменение **минорной версии** (например, c `0.0.1` на `0.1.0`) — добавление новой функции;
+- изменение **мажорной версии** (например, c `0.0.1` на `1.0.0`) — добавление функции, которая кардинально меняет возможности модуля; масштабное изменение интерфейса или завершение крупного этапа работы.
 
-Перед номером версии в теге git и контейнере registry **всегда** добавляется буква "v". Примеры: `v0.0.73`, `v1.0.0`.
+Перед номером версии в теге git и контейнере registry **всегда** добавляется буква "v". Например: `v0.0.73`, `v1.0.0`.
 
 ## Каналы обновлений
 
-Версия модуля при публикации должна *перемещаться* по [каналам обновлений](../../deckhouse-release-channels.html) от менее стабильного к более стабильному: `Alpha` -> `Beta` -> `EarlyAccess` -> `Stable` -> `RockSolid`.
+Версия модуля при публикации должна оставаться неизменной на всех [каналах обновлений](../../deckhouse-release-channels.html) от менее стабильного к более стабильному: `Alpha` → `Beta` → `EarlyAccess` → `Stable` → `RockSolid`.
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:18]:
```suggestion
При публикации новой версии модуля следует соблюдать порядок распространения по [каналам обновлений](../../deckhouse-release-channels.html) в зависимости от их стабильности: сначала на менее стабильные, затем на более стабильные, то есть от `Alpha` к `RockSolid` в следующей последовательности: `Alpha` → `Beta` → `EarlyAccess` → `Stable` → `RockSolid`.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/module-development/VERSIONING.md [hunk 9]
```diff
@@ -14,35 +14,139 @@ The git tag and registry container **always** have a "v" before the version numb
...
+<body>
+
+<table>
+    <thead>
+        <tr class="header-row">
+            <th rowspan="2">Lifecycle stage</th>
+            <th colspan="5" style="text-align:center;">Release channels</th>
+        </tr>
+        <tr class="sub-header">
+            <th>Alfa</th>
```
Comment (z9r5) [inline docs/documentation/pages/module-development/VERSIONING.md:99]:
```suggestion
            <th>Alpfa</th>
```

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 10]
```diff
@@ -6,44 +6,152 @@ lang: ru
...
+<body>
+
+<table>
+    <thead>
+        <tr class="header-row">
+            <th rowspan="2" style="text-align:center; vertical-align: middle;">Стадия жизненного цикла</th>
+            <th colspan="5" style="text-align:center; vertical-align: middle;">Каналы обновлений</th>
+        </tr>
+        <tr class="sub-header">
+            <th style="text-align:center; vertical-align: middle;">Alfa</th>
```
Comment (z9r5) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:104]:
```suggestion
            <th style="text-align:center; vertical-align: middle;">Alpfa</th>
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mortelumina98. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 11]
```diff
@@ -6,54 +6,96 @@ lang: ru
...
+            <td>Отказ от использования</td>
+        </tr>
+    </tbody>
+</table>
+
+- **Эксперименты** — Проверка функциональности, эксперименты и тестирование;
+- **Опытная эксплуатация** — Проверка функциональности, эксперименты и тестирование. Точечное использование опытными пользователями в окружениях, приравненных ; продуктивным.
+- **Ограниченная эксплуатация** — Окружения разработки, пилотные проекты, малозначимые продуктивные окружения;
+- **Промышленная эксплуатация** — Продуктивные окружения и приравненные к ним;
+- **Промышленная эксплуатация** в ответственных системах — Критично важные продуктивные окружения и приравненные к ним;
```
Comment (mortelumina98) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:91]:
```suggestion
- **Промышленная эксплуатация в ответственных системах** — Критично важные продуктивные окружения и приравненные к ним;
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/module-development/VERSIONING_RU.md [hunk 12]
```diff
@@ -6,54 +6,98 @@ lang: ru
...
+            <td>Отказ от использования</td>
+            <td>Отказ от использования</td>
+            <td>Отказ от использования</td>
+            <td>Отказ от использования</td>
+        </tr>
+    </tbody>
+</table>
+
+- **Эксперименты** — Проверка функциональности, эксперименты и тестирование;
+- **Опытная эксплуатация** — Проверка функциональности, эксперименты и тестирование. Точечное использование опытными пользователями в окружениях, приравненных ; продуктивным.
```
Comment (z9r5) [inline docs/documentation/pages/module-development/VERSIONING_RU.md:89]:
```suggestion
- **Опытная эксплуатация** — Проверка функциональности, эксперименты и тестирование. Точечное использование опытными пользователями в окружениях, приравненных к продуктивным.
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10229](https://github.com/deckhouse/deckhouse/pull/10229) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**